sdk/include/ndk/haltypes.h

```diff
@@ -276,6 +2138,8 @@ extern NTSYSAPI HAL_PRIVATE_DISPATCH HalPrivateDispatchTable;
 #define HALPRIVATEDISPATCH (&HalPrivateDispatchTable)
 #endif
 
+extern NTSYSAPI PHAL_IOMMU_DISPATCH HalIommuDispatch;
```

Comment:
Is this an export of ntoskrnl.exe or of hal.dll? And which Windows version introduced it?

Reply:
OK let's keep it for the time being, but basically if this is the name of an "internal" variable, not exposed via exports by anything, then probably it's not worth to include it here.